content/en/news/releases/1.6.x/announcing-1.6/change-notes/index.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Change Notes
+description: Istio 1.6 release notes.
+weight: 10
+---
+
+## Traffic Management
```

Comment:
Can we order by utility/relevance. Generally want use-facing API changes top of the lists (Kubernetes Service APIs, Workload Entry, etc.) then improvements, installation impacts and finally bug fixes.

Reply:
Sure. I added sections based on your feedback and then took a rough guess at placing each item.